### PR TITLE
fix: sending new position to prior position topic when transitioning

### LINKED ISSUE #576: Players seem stuck in their place when in fact they are out of sight
This happens both when the players walk away or teleport from one's position.

This is due to the other player's position not being reported as soon as it moves away and is unsubscribed from the parcel's position topic.

### PR DESCRIPTION
**Pull request by moliva**
Fixes #576 .

### CODE REVIEW

File: packages/shared/comms/index.ts [hunk 1]
```diff
@@ -35,6 +35,8 @@ import { ProfileForRenderer } from 'decentraland-ecs/src'
 type Timestamp = number
 type PeerAlias = string
 
+const UNREACHABLE_POSITION = [1000, 1000, 1000, 0, 0, 0, 0] as Position
```
Comment (eordano) [inline packages/shared/comms/index.ts]: Warning! This falls in parcel `62,62`

**Review by eordano — CHANGES_REQUESTED** [review]
`62,62` is not unreachable

File: packages/shared/comms/index.ts (hunk 1 above)
Comment (moliva) [reply packages/shared/comms/index.ts]: Good catch!

**Review by eordano — APPROVED** [review]

File: packages/shared/comms/index.ts [hunk 2]
```diff
@@ -35,6 +35,16 @@ import { ProfileForRenderer } from 'decentraland-ecs/src'
 type Timestamp = number
 type PeerAlias = string
 
+const UNREACHABLE_POSITION: Position = [
+  parcelLimits.parcelSize * 1000,
```
Comment (eordano) [inline packages/shared/comms/index.ts]:
```suggestion
  1000 * parcelLimits.parcelSize,
```
Yoda notation might help here?

**Review by eordano — APPROVED** [review]
Nit

**Pull request merged**